packages/client/tests/functional/0-legacy-ports/query-raw/tests.ts

```diff
@@ -55,7 +55,15 @@ testMatrix.setupTestSuite(
         sqlserver: [{ '': 1 }],
       }
 
-      expect(result).toStrictEqual(results[provider])
+      const resultsByDriverAdapter = {
+        js_planetscale: [{ ':vtg1 /* INT64 */': BigInt('1') }],
+      }
```

Comment:
Why do we expect this weird string as the returned key? Is this a normal thing for the serverless driver? Does this only affect queries like `SELECT 1` or also other, normal queries that would return some data from columns?

Reply:
I assume it's a Vitess thing, and we would have to do the same if we ran tests with Vitess, but let's wait for @jkomyno to confirm.

Reply:
Good point that we are not running Vitess tests, so we have no real comparison.

Reply:
Either way, this is not something we control, that's just how it is regardless of whether this comes from Vitess or from the serverless driver. This only affects the raw queries that return values without column names, otherwise normal Prisma queries would not be able to work.

There's also no universally expected behaviour in this case, it is different for each native connector too (Postgres returns `{ '?column?': 1 }` for this query, MySQL without Vitess returns `{ '1': 1 }`, SQL Server returns `{ '': 1 }`).